website/content/en/docs/building-operators/golang/tutorial.md

```diff
@@ -92,12 +78,28 @@ controllers/memcached_controller.go
 
 This will scaffold the Memcached resource API at `api/v1alpha1/memcached_types.go` and the controller at `controllers/memcached_controller.go`.
 
-See the [API terminology doc][api_terms_doc] for details on the CRD API conventions.
+**Note:** This guide will cover the default case of a single group API. If you would like to support Multi-Group APIs see the [Single Group to Multi-Group][multigroup-kubebuilder-doc] doc.
 
-To understand the API Go types and controller scaffolding see the Kubebuilder [api doc][kb_api_doc] and [controller doc][kb_controller_doc].
+#### Understanding API’s
+
+The goal of this command is to create Custom Resource (CR) and Custom Resource Definition (CRD) resources for the Memcached Kind. This command is creating the API with the group `cache.example.com`, and version `v1alpha1`  which uniquely identifies the new CRD of the Memcached Kind. For more info see [Extend the Kubernetes API with CustomResourceDefinitions][[kubernetes-extend-api]]. 
```

Comment:
Is there a reason 'Kind' is capitalized here? It's not a kube term of art I'm unfamiliar with, is it? maybe "Memcached type"? also the link markdown is broken

Reply:
Kind is the k8s nomenclature, see: 

```
apiVersion: cache.example.com/v1alpha1
kind: Memcached
```

To create the API for the Kind (which is a type as you could check) we run the following  command; 

create api --group --version --kind

I hope that clarifies.

Reply:
got it. link is still broken tho :+1: